utils/rpms/daos.spec

```diff
@@ -226,11 +226,19 @@ This is the package needed to run a DAOS client
 
 %package tests
 Summary: The entire DAOS test suite
-Requires: %{name}-client-tests-openmpi%{?_isa} = %{version}-%{release}
+Requires: %{name}-client-tests%{?_isa} = %{version}-%{release}
```

Comment:
I'm not sure that you can do this all in one commit.  This will cause testing on this branch to break until the pipeline-lib change lands.  Unless you close landing on this branch after this until the pipeline-lib update lands.

Reply:
I'm temporarily including installing the `daos-client-tests-openmpi` package through the Jenkins file until packaging in all branches generates a `daos-test-internal` package, at which point pipeline-lib can be updated to install it instead of `daos-tests`.